streaming/base/format/xsv/writer.py

```diff
@@ -201,6 +207,7 @@ class TSVWriter(XSVWriter):
             Defaults to ``None``.
         size_limit (int, optional): Optional shard size limit, after which point to start a new
             shard. If None, puts everything in one shard. Defaults to ``None``.
+        **kwargs (Any): Additional settings for the Writer
```

Comment:
trailing .

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.